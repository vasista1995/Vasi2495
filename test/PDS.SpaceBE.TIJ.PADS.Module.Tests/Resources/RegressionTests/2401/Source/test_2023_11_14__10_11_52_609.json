{
  "IdSource": "SPACEACT2:TIJ:BE:104OOFV.1:Pre_assy_Pasivacion:SP:PasivationAperture_DistanceY:1433:3096850:0",
  "Data_flat_Metadata": {
    "Segment": "Tijuana",
    "Module": "SP",
    "ParameterClass": "PPAP",
    "ProcessEquipment": "WEOA0102",
    "Category": "PBF",
    "Classification": "SPC-Monitoreo",
    "Operator": "cerna",
    "Layout": "8IN_ELLEMEX",
    "PartNumber": "913520DH",
    "Area": "Pre_assy_Pasivacion",
    "Lot": "104OOFV.1",
    "ParameterName": "PasivationAperture_DistanceY",
    "ParameterUnit": "mm",
    "SiteKey": "TIJ",
    "ChannelId": "1433",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-10T02:51:32Z",
    "CreatedTimestamp": "2023-10-09T19:51:32Z",
    "UpdatedTimestamp": "2023-10-09T19:51:32Z",
    "UpdatedTimestampUtc": "2023-10-10T02:51:32Z",
    "ExportedTimestampUtc": "2023-10-09T19:51:32Z",
    "ExportedTimestamp": "2023-10-09T19:51:32Z",
    "SampleTimestampUtc": "2023-10-10T02:51:32Z",
    "SampleTimestamp": "2023-10-09T19:51:32Z",
    "SourceDataLevel": "C",
    "Equipment": "MSNI0001",
    "ChannelName": "PasivationAperture_DistanceY/WDSON/PBF/8IN_ELLEMEX",
    "RvStoreFlag": "Y",
    "CalcCreated": "10/05/2018 09:37:08",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 2291,
    "ViolCount": 0,
    "CalculationStrategy": "ND",
    "LdsID": "2401",
    "IsCalcParam": 0,
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 0.624,
        "SpecLow": 0.594,
        "SpecTarget": 0.609,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNYNNYNYYYYYYYYYYYNNNYYYYYYYNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": ",6056",
        "Median": ",606",
        "Sigma": ",00167332005306815",
        "Min": ",604",
        "Max": ",608",
        "Q1": ",604",
        "Q3": ",606"
      },
      "ControlLimits": {
        "ctrl_mean_low": 0.59446,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 0.60281,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 0.61116,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "Y",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "Y",
        "CtrlSigmaHighEnabled": "Y",
        "ctrl_range_low": 0.0,
        "CtrlRangeLowEnabled": "N",
        "ctrl_range_target": 0.005565,
        "ctrl_range_high": 0.01113,
        "CtrlRangeHighEnabled": "N",
        "ExtMaMV": 0.604514285714286,
        "ExtEwmaMV": 0.604627122506961,
        "ExtMsMV": 0.000141421356237372,
        "EwmaS": 0.00179916026460533,
        "EwmaR": 0.00425368962545698,
        "ExtEwmaMVLCL": 0.0,
        "ExtEwmaMVCenter": 0.0,
        "ExtEwmaMVUCL": 0.0,
        "ExtMaMVLCL": 0.0,
        "ExtMaMVCenter": 0.0,
        "ExtMaMVUCL": 0.0,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.0,
        "ExtMSMVCenter": 0.0,
        "ExtMSMVUCL": 0.0,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 0.0,
        "RawUal": 0.0,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 3096850,
        "Seqnr": 2,
        "Value": 0.608,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096850,
        "Seqnr": 1,
        "Value": 0.606,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096850,
        "Seqnr": 5,
        "Value": 0.604,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096850,
        "Seqnr": 4,
        "Value": 0.606,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      },
      {
        "SampleId": 3096850,
        "Seqnr": 3,
        "Value": 0.604,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 5
  },
  "ProductionAction": {
    "Id": "SPACEACT1:TIJ:BE:Pre_assy_Pasivacion:SP:PasivationAperture_DistanceY:1433",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "Measlot:104OOFV.1",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "TIJ:BE:104OOFV.1:Pre_assy_Pasivacion:SP:PasivationAperture_DistanceY:1433:3096850:0"
  }
}