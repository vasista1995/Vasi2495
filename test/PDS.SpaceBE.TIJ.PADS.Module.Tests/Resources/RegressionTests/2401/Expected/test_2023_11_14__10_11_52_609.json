{
  "Data_flat_Metadata": {
    "Classification": "SPC-Monitoreo",
    "Layout": "8IN_ELLEMEX",
    "PartNumber": "913520DH",
    "Area": "Pre_assy_Pasivacion",
    "MeasLot": "104OOFV.1",
    "SiteKey": "TIJ",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-10-10T02:51:32Z",
    "BeginTimestamp": "2023-10-09T19:51:32Z",
    "EndTimestampUtc": "2023-10-10T02:51:32Z",
    "EndTimestamp": "2023-10-09T19:51:32Z"
  },
  "Data_flat_StrangeMetadata": {
    "Module": "SP",
    "Category": "PBF",
    "Operator": "cerna"
  },
  "Data_1list_Parameters": [
    {
      "Data_1list_RawValues": [
        {
          "process_equipment": "WEOA0102",
          "Seqnr": 2,
          "SampleId": 3096850,
          "Value": 0.608,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-09T19:51:32Z",
          "SampleTimestampUtc": "2023-10-10T02:51:32Z",
          "CreatedTimestamp": "2023-10-09T19:51:32Z",
          "CreatedTimestampUtc": "2023-10-10T02:51:32Z",
          "UpdatedTimestamp": "2023-10-09T19:51:32Z",
          "UpdatedTimestampUtc": "2023-10-10T02:51:32Z",
          "NumViolations": 0,
          "SampleMean": 0.6056,
          "SampleMin": 0.604,
          "SampleMax": 0.608,
          "SampleMedian": 0.606,
          "SampleStdev": 0.00167332005306815,
          "SampleQ1": 0.604,
          "SampleQ3": 0.606,
          "SampleSize": 5
        },
        {
          "process_equipment": "WEOA0102",
          "Seqnr": 1,
          "SampleId": 3096850,
          "Value": 0.606,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-09T19:51:32Z",
          "SampleTimestampUtc": "2023-10-10T02:51:32Z",
          "CreatedTimestamp": "2023-10-09T19:51:32Z",
          "CreatedTimestampUtc": "2023-10-10T02:51:32Z",
          "UpdatedTimestamp": "2023-10-09T19:51:32Z",
          "UpdatedTimestampUtc": "2023-10-10T02:51:32Z",
          "NumViolations": 0,
          "SampleMean": 0.6056,
          "SampleMin": 0.604,
          "SampleMax": 0.608,
          "SampleMedian": 0.606,
          "SampleStdev": 0.00167332005306815,
          "SampleQ1": 0.604,
          "SampleQ3": 0.606,
          "SampleSize": 5
        },
        {
          "process_equipment": "WEOA0102",
          "Seqnr": 5,
          "SampleId": 3096850,
          "Value": 0.604,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-09T19:51:32Z",
          "SampleTimestampUtc": "2023-10-10T02:51:32Z",
          "CreatedTimestamp": "2023-10-09T19:51:32Z",
          "CreatedTimestampUtc": "2023-10-10T02:51:32Z",
          "UpdatedTimestamp": "2023-10-09T19:51:32Z",
          "UpdatedTimestampUtc": "2023-10-10T02:51:32Z",
          "NumViolations": 0,
          "SampleMean": 0.6056,
          "SampleMin": 0.604,
          "SampleMax": 0.608,
          "SampleMedian": 0.606,
          "SampleStdev": 0.00167332005306815,
          "SampleQ1": 0.604,
          "SampleQ3": 0.606,
          "SampleSize": 5
        },
        {
          "process_equipment": "WEOA0102",
          "Seqnr": 4,
          "SampleId": 3096850,
          "Value": 0.606,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-09T19:51:32Z",
          "SampleTimestampUtc": "2023-10-10T02:51:32Z",
          "CreatedTimestamp": "2023-10-09T19:51:32Z",
          "CreatedTimestampUtc": "2023-10-10T02:51:32Z",
          "UpdatedTimestamp": "2023-10-09T19:51:32Z",
          "UpdatedTimestampUtc": "2023-10-10T02:51:32Z",
          "NumViolations": 0,
          "SampleMean": 0.6056,
          "SampleMin": 0.604,
          "SampleMax": 0.608,
          "SampleMedian": 0.606,
          "SampleStdev": 0.00167332005306815,
          "SampleQ1": 0.604,
          "SampleQ3": 0.606,
          "SampleSize": 5
        },
        {
          "process_equipment": "WEOA0102",
          "Seqnr": 3,
          "SampleId": 3096850,
          "Value": 0.604,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-09T19:51:32Z",
          "SampleTimestampUtc": "2023-10-10T02:51:32Z",
          "CreatedTimestamp": "2023-10-09T19:51:32Z",
          "CreatedTimestampUtc": "2023-10-10T02:51:32Z",
          "UpdatedTimestamp": "2023-10-09T19:51:32Z",
          "UpdatedTimestampUtc": "2023-10-10T02:51:32Z",
          "NumViolations": 0,
          "SampleMean": 0.6056,
          "SampleMin": 0.604,
          "SampleMax": 0.608,
          "SampleMedian": 0.606,
          "SampleStdev": 0.00167332005306815,
          "SampleQ1": 0.604,
          "SampleQ3": 0.606,
          "SampleSize": 5
        }
      ],
      "process_equipment": "WEOA0102",
      "ParameterClass": "PPAP",
      "Segment": "Tijuana",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "Y",
          "SpecLowEnabled": "Y",
          "spec_high": 0.624,
          "spec_low": 0.594,
          "target": 0.609
        },
        "ControlLimits": {
          "ctrl_mean_low": 0.59446,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 0.60281,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 0.61116,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "Y",
          "CtrlSigmaCenterEnabled": "Y",
          "CtrlRangeCenterEnabled": "N",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "Y",
          "ctrl_range_low": 0.0,
          "CtrlRangeLowEnabled": "N",
          "ctrl_range_target": 0.005565,
          "ctrl_range_high": 0.01113,
          "CtrlRangeHighEnabled": "N"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 5,
        "base_count": 5,
        "flagged_count": 0,
        "mean": 0.6056,
        "median": 0.606,
        "stdev": 0.0014966629547095778,
        "range": 0.0040000000000000036,
        "min": 0.604,
        "max": 0.608,
        "q02": 0.604,
        "q05": 0.604,
        "q25": 0.604,
        "q75": 0.606,
        "q95": 0.6076,
        "q98": 0.60784,
        "NumViolations": 0,
        "Samples": "3096850"
      },
      "ParameterName": "PasivationAperture_DistanceY",
      "ParameterUnit": "mm",
      "ChannelId": "1433",
      "SourceDataLevel": "C",
      "channel_name": "PasivationAperture_DistanceY WDSON PBF 8IN_ELLEMEX",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-10-10T02:51:32Z",
      "created_timestamp": "2023-10-09T19:51:32Z",
      "updated_timestamp_utc": "2023-10-10T02:51:32Z",
      "updated_timestamp": "2023-10-09T19:51:32Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-10-10T02:51:32Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED2:TIJ:BE:Pre_assy_Pasivacion:SP",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "Measlot:104OOFV.1",
    "IdSystemName": "Measlot",
    "Type": "Lot"
  },
  "SystemLog": {
    "UniqueSourceKey": "TIJ:BE:104OOFV.1:Pre_assy_Pasivacion:SP:PasivationAperture_DistanceY:1433:3096850:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-10-09T19:51:32Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "Measlot:104OOFV.1:SPACEAGGED2:TIJ:BE:Pre_assy_Pasivacion:SP:ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "TIJ"
  }
}